src/Shouldly.Tests/Shouldly.Tests.csproj

```diff
@@ -19,7 +18,6 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
     <PackageReference Include="PublicApiGenerator" Version="8.0.1" />
     <PackageReference Include="TestStack.ConventionTests" Version="3.0.1" />
-    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
```

Comment:
`dotnet-xunit` was discontinued: `https://github.com/xunit/xunit/issues/1835#issuecomment-431199286`

Version 2.3.1 has no rollforward policy and can only run on .NET Core 2.0 which is EOL.